e2e/removed-and-added-files-collector/expected-output.diff

```diff
@@ -0,0 +1,7 @@
+! [NOTE] File
+ !        "./removed-and-added-files-c
+ !        ollector/test" will be added
+
+
+
+ [OK] Rector is done!
```

Comment:
this `[OK] Rector is done!` printed message should actually not exists, as it is running with `--dry-run`, but probably need in separate PRs for improvement :)